tools/toolchains/arm.py

```diff
@@ -233,6 +233,11 @@ def link(self, output, objects, libraries, lib_dirs, scatter_file):
         cmd_pre = self.ld + args
         cmd = self.hook.get_cmdline_linker(cmd_pre)
 
+        # Create Secure library
+        if self.target.core == "Cortex-M23" or self.target.core == "Cortex-M33":
```

Comment:
Moving this here breaks exporters: The exporters have to know to append these options somehow, and this location prevents them from being able to.